server/src/main/scala/ch.epfl.scala.index.server/routes/api/SearchApi.scala

```diff
@@ -201,16 +212,35 @@ class SearchApi(
               optionalSession(refreshable, usingCookies) { userId =>
                 val user = session.getUser(userId)
                 searchParams(user) { params =>
-                  complete {
-                    autocomplete(params)
-                  }
+                  val autoCompletion = autocomplete(params)
+                  complete(autoCompletion)
                 }
               }
             }
           }
       }
     }
 
+  private def convertProject(
+      cli: Boolean
+  )(project: Project): SearchApi.Project = {
+    val artifacts = if (cli) project.cliArtifacts.toList else project.artifacts
+    SearchApi.Project(
```

Comment:
Can we also include defaultArtifact since search API under the hood redirects to it anyway.